app/ledger.js

```diff
@@ -383,6 +391,9 @@ eventStore.addChangeListener(() => {
 var initialize = (onoff) => {
   enable(onoff)
 
+  // Check if relevant browser notifications should be shown every 15 minutes
+  setInterval(showNotifications, 15 * msecs.minute)
```

Comment:
`clearInterval` never gets called on this, so i think a new timer is added every time the user clicks the on/off switch. this may lead to notifications being shown more often than intended or something